scripts/p2p-test-tool.py

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env python3
+#!/usr/bin/env python3.6
```

Comment:
This fails if there's only `/usr/bin/python3.7`. I guess you could just let it fail on syntax error with `python3` if it's Python < 3.6. Anyway, this only runs on Ubuntu VMs, so it doesn't matter much.